histomicsui/web_client/panels/DrawWidget.js

```diff
@@ -725,6 +770,21 @@ var DrawWidget = Panel.extend({
         }
     },
 
+    countPixelmap(pixelmap, operation) {
+        let toChange = {};
+        for (let ix = 0; ix < pixelmap.attributes.values.length / 2; ix++) {
```

Comment:
I think the `/ 2` is only required if `pixelmap.get('boundaries')` is truthy.

Reply:
Okay- see 5bc64c0.